### ISSUE TITLE
Bazel: Run Bazel build on the CircleCi

### ISSUE THREAD

**Issue by lukasholzer** [dynatrace-oss/barista#1018]
# Feature Request

Run bazel build on the circle CI in an extra workflow that is running in parallel with the `pr-check` workflow that we do not affect current development.

This workflow should reuse the dependency install job and add another job for the `bazel build //...` commands. 

This workflow should only run with a `branch-filter` for all branches that are not forks.
Exclude all `pull/${number}` branches from building.

**Comment by lukasholzer** [comment]
This is already done. Furthermore, our future Plans are not to run bazel on circleci as the boostrapping time of bazel in a cold docker container takes forever. So we will switch to Buildkite for that.
